tests/test_plugin.py

```diff
@@ -4415,3 +4415,10 @@ def test_sql_crash(node_factory, bitcoind):
 
     assert 'updates' not in only_one(l1.rpc.listpeerchannels()['channels'])
     l1.rpc.sql(f"SELECT * FROM peerchannels;")
+
+
+def test_listchannels_broken_message(node_factory):
+    """This gave a bogus BROKEN message with deprecated-apis enabled"""
+    l1 = node_factory.get_node(options={'allow-deprecated-apis': True})
+
+    l1.rpc.listchannels()
```

Comment:
We may want to assert on the logs?

Reply:
Cleanup already does this automatically...